### ISSUE TITLE
move baseline parallelization infrastructure to hera_qm

### ISSUE THREAD

**Issue by anguta** [HERA-Team/hera_cal#614]
full day waterfall metrics would be a useful option for rfi flagging so I think some of the machinery for doing this should be moved to `hera_qm` (a dependency for `hera_cal`).

**Comment by jsdillon** [comment]
I don't think this is a great idea... hera_cal and hera_qm have a complex interdependency... it's not that simple.

Also, full-day metrics are very fast because they operate on single waterfalls, not per-baseline.

**Comment by anguta** [comment]
My understanding is that the metrics are calculated per-baseline over limited time windows and then combined into a waterfall. The functionality that I am proposing is to do the same thing but calculate the metrics over full day time windows.

**Comment by anguta** [comment]
I agree that we don't need to move stuff around. The functionality needed for this only requires one short redundant function.

**Comment by jsdillon** [comment]
I see. Well, if we were planning to do that, we wouldn't need all the chunking we just built... This doesn't feel like a necessary upgrade for the near future... Let's see how fast the corner turn actually is to figure out whether or not this would save any time.
